addons/hr_holidays/static/src/components/thread_view/thread_view.xml

```diff
@@ -3,7 +3,7 @@
     <t t-inherit="mail.ThreadView" t-inherit-mode="extension">
         <xpath expr="//*[@name='loadingCondition']" position="before">
             <t t-if="threadView.thread.correspondent and threadView.thread.correspondent.out_of_office_date_end">
```

Comment:
condition here could be changed on `outOfOfficeText` instead of `out_of_office_date_end` now